apps/server/src/modules/tool/common/uc/tool-permission-helper.ts

```diff
@@ -1,16 +1,21 @@
 import { forwardRef, Inject, Injectable } from '@nestjs/common';
-import { EntityId, LegacySchoolDo, User } from '@shared/domain';
-import { AuthorizableReferenceType, AuthorizationContext, AuthorizationService } from '@src/modules/authorization';
+import { Course, EntityId, LegacySchoolDo, User } from '@shared/domain';
+import { AuthorizationContext, AuthorizationService } from '@src/modules/authorization';
 import { LegacySchoolService } from '@src/modules/legacy-school';
+import { CourseRepo } from '@shared/repo';
 import { ContextExternalTool } from '../../context-external-tool/domain';
 import { SchoolExternalTool } from '../../school-external-tool/domain';
-import { ContextTypeMapper } from '../mapper';
+// import { ContextTypeMapper } from '../mapper';
```

Comment:
Hint should be stay in for the team, i have disabled the mapping to make it possible to replace the code for now.